common/artifacts/902_wc_artifacts_crown_crowns.txt

```diff
@@ -15,4 +15,26 @@ crown_of_trollbanes = {
 	picture = "GFX_inv_crown_02"
 
 	slot = crown
-}
\ No newline at end of file
+}
+#Crown Of Lordearon
+crown_of_menethils = {
+	quality = 3  #Rare
```

Comment:
I'm still no fan of making it as rare/high tier as the crown of the trollbanes, whose dynasty lasted for a way longer time when the game starts. The stats you chose for it however are good, i think, since terenas was a good diplomat but not much more :)